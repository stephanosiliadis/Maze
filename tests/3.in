17 20
1 7 14 10 -1
2 1 11
2 11 4
3 2 8
4 3 1
5 4 7
6 5 5
7 6 23
7 8 2
9 8 9
9 17 0
11 12 7
12 13 4
13 14 5
14 15 2
15 14 10
15 16 3
16 10 10
16 15 1
10 16 3
10 9 0
