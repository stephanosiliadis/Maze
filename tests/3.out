133
1 R
2
3
4
5
6
7 R
6
5
4
3
2
11
12
13
14 R
15
16
10 R
9
17
